tests/flows/erc20_to_native_token_bridge.go

```diff
@@ -59,45 +61,66 @@ func ERC20ToNativeTokenBridge(network interfaces.LocalNetwork) {
 	Expect(err).Should(BeNil())
 	log.Info("Example ERC20 Contract Address: " + exampleERC20ContractAddress.Hex())
 
-	// Deploy the contracts
-	// Both contracts in this test will be deployed to 0x3405506b3711859c5070949ed9b700c7ba7bf750,
-	// though they do not necessarily have to be deployed at the same address, each contract needs
-	// to know the address of the other.
-	// The nativeTokenDestination contract must be added to "adminAddresses" of "contractNativeMinterConfig"
-	// in the genesis file for the subnet. This will allow it to call the native minter precompile.
-	erc20TokenSourceAbi, err := erc20tokensource.ERC20TokenSourceMetaData.GetAbi()
-	Expect(err).Should(BeNil())
-	utils.DeployContract(
-		ctx,
-		ERC20TokenSourceByteCodeFile,
-		deployerPK,
-		sourceSubnet,
-		erc20TokenSourceAbi,
-		sourceSubnet.TeleporterRegistryAddress,
-		destSubnet.BlockchainID,
-		bridgeContractAddress,
-		exampleERC20ContractAddress,
-	)
+	{
+		// Fund the deployer addresses on each chain
+		utils.SendNativeTransfer(
+			ctx,
+			sourceSubnet,
+			fundedKey,
+			deployerAddress,
+			utils.BigIntMul(initialReserveImbalance, big.NewInt(2)),
```

Comment:
Is there a reason why we're using `initialReserveImbalance*2` here and below? IIUC, we should be providing the amount needed to deploy the contract, which is likely unrelated to the reserve imbalance. If so, can we update to a new variable/name?

Reply:
We also use the deployer address to send tokens from one chain to another. I don't want to use the `funded` address to keep this test independent of the others, and using a different address would add more code. The amount transferred in the test is 1.25*`initialReserveImbalance`, so 2* accounts for contract deployment and gas fees/headroom.

Reply:
> The amount transferred in the test is 1.25*initialReserveImbalance, so 2* accounts for contract deployment and gas fees/headroom.

Gotcha, thanks for the explanation. For the ERC20->native case it's an independent asset though right? So we only need the amount to deploy the contract?

Also could you add that explanation as a comment in the native->native cases? I think it's very helpful context for who/whenever is looking at these down the line 👍